src/editor/app.js

```diff
@@ -712,12 +723,17 @@ $(document).ready(function () {
         UiDriver.sendMessage("J_EVT_CLEAN_CHANGED", isCleanOrForced(changeGeneration));
     });
 
-    editor.on("cursorActivity", function(instance, changeObj) {
-        UiDriver.sendMessage("J_EVT_CURSOR_ACTIVITY");
+    editor.on("cursorActivity", function(instance) {
+        UiDriver.sendMessage("J_EVT_CURSOR_ACTIVITY", getActivityInfo());
     });
 
     editor.on("focus", function() {
         UiDriver.sendMessage("J_EVT_GOT_FOCUS");
+        UiDriver.sendMessage("J_EVT_CURSOR_ACTIVITY", getActivityInfo());
+    });
+
+    $(document).mouseenter(function() {
+        UiDriver.sendMessage("J_EVT_CURSOR_ACTIVITY", getActivityInfo());
```

Comment:
Why's this necessary? Isn't this basically the "on hover"-event?

Reply:
CodeMirror doesn't have a native onHover event so this is the next best thing available, unfortunately.

Otherwise the status text would remain blank until the user moves the cursor to a different postiion.

Reply:
I mean, why is it necessary at all to have this? The `"J_EVT_CURSOR_ACTIVITY"` signal is also emitted on focus; I'd assume that ought to be enough.

Reply:
Hm is it possible to use `CustomWebView::gotFocus()` instead? That one should work fine.

Reply:
That's not a bad idea... there's a few things I might have to change to get it to work but its likely the "proper" way to do it assuming it works.

I've noticed QWebEngine loves to eat events so the typical standard "it should work" might not apply here.  We'll see, however.

Reply:
I was thinking bring back `refreshEditorUiCursorInfo` and call it through `on_cursorActivity` and `on_currentTabChanged` or whatever the proper signal is. Pretty sure we're already handling CustomWebView's focus event somewhere in MainWindow.